vfp/WVFPDP-02.DATA

```diff
@@ -0,0 +1,333 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2023 Equinor
+
+-- This test case is testing keyword WVFPDP (adjusting bottom hole pressure from vfp). This case includes 
+-- one producer with aquifer support. The producer is using artificial lift last 60 days of the simulation.  
+\-- Item 2 in keyword WVFPDP is set to -20.0 bars and item 3 (loss scaling factor) is set to 0.75
```

Comment:
This backslash character at the beginning of the line looks a little strange to me.  Is that really supposed to be a valid comment character?

Reply:
tanks, my bad. I keystroke that ended up in a wrong window. 

This is not a valid line and Flow will give an error message 
```
String \ not formatted as valid keyword
```